packages/babel-types/src/definitions/typescript.js

```diff
@@ -494,7 +494,7 @@ defineType("TSTypeParameterDeclaration", {
 });
 
 defineType("TSTypeParameter", {
-  visitor: ["constraint", "default"],
+  visitor: ["constraint", "default", "name"],
```

Comment:
Only nodes can be visited, so we can't add `name` to `visitor`.
You can define `builder: ["constraint", "default", "name"],` instead.